mapie/estimator/estimator.py

```diff
@@ -152,6 +152,7 @@ class EnsembleRegressor(EnsembleEstimator):
         "single_estimator_",
         "estimators_",
         "k_",
+        "use_split_method",
```

Comment:
Why are we adding this in the `fit_attributes`. I feel like this might create problems for people that have a custom algorithm.

Reply:
There won't be any problems with custom algo :)